src/main/resources/crafter/engine/studio-server-config.properties

```diff
@@ -1,308 +1,325 @@
+# Defines maximum number of active database connections in database connection pool.
 studio.db.pool.maxActiveConnections=100
-# maximum number of idle database connections to retain in pool
+# Defines maximum number of idle database connections to retain in database connection pool.
 studio.db.pool.maxIdleConnections=30
-# minimum number of idle database connections to retain in pool
+# Defines minimum number of idle database connections to retain in database connection pool.
 studio.db.pool.minIdleConnections=10
-# maximum time to wait for a database connection to become available in ms
+# Defines maximum waiting time for a database connection to become available (value is in milliseconds).
 studio.db.pool.maxWaitTime=10000
-# database initializer
+# Turn database initializer on (value true) or off (value false).
 studio.db.initializer.enabled=false
-# if database connections retrieved from the pool should be tested before being used
+# Test database connections retrieved from the pool before being used (value true). To disable database connection testing before being used set value to false.
 studio.db.testOnBorrow=true
-# the query to validate the connection for MS SQL Server
+# The SQL query to validate the connection for MS SQL Server.
 studio.db.validationQuery.sqlserver=SELECT 1
-# the query to validate the connection for Postgres
+# The SQL query to validate the connection for Postgres.
 studio.db.validationQuery.postgres=SELECT 1
-# the query to validate the connection for MySQL
+# The SQL query to validate the connection for MySQL.
 studio.db.validationQuery.mysql=SELECT 1
-# the query to validate the connection for Oracle
+# The SQL query to validate the connection for Oracle
 studio.db.validationQuery.oracle=SELECT 1 FROM DUAL
-# the query to validate the connection for Derby
+# The SQL query to validate the connection for Derby
 studio.db.validationQuery.derby=SELECT 1 FROM SYSIBM.SYSDUMMY1
-# the query to validate the connection for DB2
+# The SQL query to validate the connection for DB2
 studio.db.validationQuery.db2=SELECT 1 FROM SYSIBM.SYSDUMMY1
-# the interval to validate the query
+# Time interval to validate the database connection with SQL query (value is in milliseconds)
 studio.db.validationInterval=34000
 
 ########################
 # Environment settings #
 ########################
-# Environment configuration (environment overrides)
+# Defines name used for environment specific configuration. It is used for environment overrides in studio. Default value is local.
 environment=local
 environmentConfig.environment=local
 
 ############################
 # Repository configuration #
 ############################
-# Repository type (valid values: default, alfresco, alfrescoext)
-repositoryType=git
-# Security provider (valid values: default, alfresco, alfrescoext)
-securityType=db
-# Disk repository root path
+# Defines repository type used for storing content. Possible values:
+#   default (disk repository)
```

Comment:
default (default currently evaluates to disk)
disk
alfrescoext

Do not give the alfresco (non-ext) option because we do not want people using it or being confused